docs/basic-features/image-optimization.md

```diff
@@ -79,7 +79,7 @@ function Home() {
 }
 ```
 
-For dynamic images you have to provide `width`, `height` and `blurDataURL` manually.
+For dynamic or remote images, you'll have to provide `width`, `height` and `blurDataURL` manually. Solutions such as [Plaiceholder](https://github.com/joe-bell/plaiceholder) can help with this.
```

Comment:
I don't think this is the correct place to link to it. It should be in the `blurDataURL` section with the other links.

Also these references here should probably link to the anchors on this pageL width/height/blurDataURL

Reply:
✅